layouts/partials/nav.html

```
@@ -1,8 +1,24 @@
+    {{ end }}
+    <a href="{{ .URL | relLangURL }}"{{ if $active }} class="active" aria-current="page"{{ end }}>{{ .Name }}</a>
 {{ end }}
-<a href='{{ absURL ("index.xml" | relLangURL) }}'
+<a class="nav-rss" href='{{ absURL ("index.xml" | relLangURL) }}'
     data-tinylytics-event="feed.subscribe"
     data-tinylytics-event-value="main-rss"
     >{{ i18n "rss" | default "RSS" }}</a
->
\ No newline at end of file
+>
+<button
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)